src/Command/MailerCitizenProjectValidatedSummaryCommand.php

```diff
@@ -0,0 +1,155 @@
+use AppBundle\Repository\CitizenProjectRepository;
+use Doctrine\ORM\EntityManager;
+use League\Flysystem\Exception;
+use Symfony\Bridge\Monolog\Logger;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Twig\Environment;
+
+class MailerCitizenProjectValidatedSummaryCommand extends ContainerAwareCommand
+{
+    public const COMMAND_NAME = 'app:mailer:citizen-project-summary';
+    private const CITIZEN_PROJECTS_LIST_LIMIT = 15;
+
+    /** @var EntityManager */
+    private $entityManager;
+
+    /** @var MailerService */
+    private $mailer;
+
+    /** @var Logger */
+    private $logger;
+
+    /** @var Environment */
+    private $twig;
+
+    /** @var AdherentRepository */
+    private $adherentRepository;
+
+    /** @var CitizenProjectRepository */
+    private $citizenProjectRepository;
+
+    protected function configure()
+    {
+        $this
+          ->setName(self::COMMAND_NAME)
+          ->addOption('maxAge', 'ma', InputOption::VALUE_OPTIONAL, 'Max age of the citizen projects in days.')
+          ->setDescription('Sending a summary email of validated citizen projects to adherents.');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $container = $this->getContainer();
+
+        $this->entityManager = $container->get('doctrine.orm.entity_manager');
+        $this->mailer = $container->get('app.mailer.transactional');
+        $this->logger = $container->get('logger');
+        $this->twig = $container->get('twig');
+
+        $this->adherentRepository = $this->entityManager->getRepository(Adherent::class);
+        $this->citizenProjectRepository = $this->entityManager->getRepository(CitizenProject::class);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $adherents = $this->adherentRepository->findAllEnabledAdherents();
+
+        $counter = 0;
+        foreach ($adherents as $chunk) {
+            /** @var Adherent $adherent */
+            $adherent = $chunk[0];
+
+            // Finds all citizen projects near the Adherent
+            $citizenProjects = $this->citizenProjectRepository->findNearByCitizenProjectSummariesForAdherent(
+                $adherent,
+                self::CITIZEN_PROJECTS_LIST_LIMIT,
+                $input->getOption('maxAge')
+            );
+
+            if (0 === count($citizenProjects)) {
+                continue;
+            }
+
+            // Broadcast the citizen projects and returns them formatted in array
+            $broadcastedCitizenProjects = $this->broadcast($citizenProjects);
+
+            // Sends the message
+            $message = $this->createMessage($broadcastedCitizenProjects, $adherent);
+            $this->mailer->sendMessage($message);
+
+            $this->logger->info(
```

Comment:
I would move the message handling in `broadcast` so this private method could easily be replaced by a service.